assets/src/scss/editorStyle.scss

```diff
@@ -22,3 +22,6 @@
 @import "layout/gravity-forms";
 
 @import "editorOverrides";
+
+// CSS variables
+@import "base/css-variables";
```

Comment:
And we don't need to import that to [blocks.scss](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/main/assets/src/styles/blocks.scss) too?

Reply:
Hm.. testing locally it seems that it's not needed.

Reply:
I think that the editor always loads `editorStyle.scss`, so that's enough to initialise the CSS variables that are used in the blocks